### DISCUSSION TITLE
如何引入地理坐标系

### DISCUSSION THREAD

**Discussion by hanfengcan** [antvis/G6#5835] [Q&A]
如果想要使用经纬度作为节点的位置，背景是地图。有什么思路可以实现吗

**Comment by Aarebecca** [comment]
~~使用经纬度作为节点的位置~~ ➡️ 将经纬度坐标转换为 G6 画布坐标

> G6 5.0 提供了相关坐标转换的 [API](https://g6-next.antv.antgroup.com/api/graph/method#graphgetcanvasbyclientpoint)

地图背景指的是图片还是其他的地图工具库

如果是地图工具库，还需要解决缩放、拖拽交互过程中画布同步的问题
